modules/memberships/class-jetpack-memberships.php

```diff
@@ -0,0 +1,291 @@
+<?php
+/**
+ * WARNING: This file is distributed verbatim in Jetpack.
+ * There should be nothing WordPress.com specific in this file.
+ *
+ * @package memberships
+ * @hide-in-jetpack
+ */
+
+/**
+ * Class Jetpack_Memberships
+ * This class represents the Memberships functionality.
+ */
+class Jetpack_Memberships {
+	/**
+	 * CSS class prefix to use in the styling.
+	 *
+	 * @var string
+	 */
+	static public $css_classname_prefix = 'jetpack-memberships';
+	/**
+	 * Increase this number each time there's a change in CSS or JS to bust cache.
+	 *
+	 * @var string
+	 */
+	static private $version = '0.03';
```

Comment:
Could we use `JETPACK__VERSION` here instead?

Reply:
Done!